ClientApp/viking-vault/src/UserDashboard/TransactionList.css

```diff
@@ -10,6 +10,13 @@
     font-size: 1vw;
 }
 
+.transactions table td.details {
+    max-width: 150px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
 .transactions h5 {
```

Comment:
It's best to avoid using such specific html elements in your CSS selectors such as `.h5`, it would be better to replace this with a class